### PR TITLE
Make everything AGPLv3-compliant

### PR DESCRIPTION
**Pull request by ShellyHerself**
Only code change in here should be the added about command

### CODE REVIEW

File: codegen/generate.pl [hunk 1]
```diff
@@ -1,8 +1,18 @@
-#!/usr/bin/perl
```
Comment (Mimickal) [inline codegen/generate.pl:1]: You can keep the shebang here and not mess with the license

File: vulpes/network/foxnet/effect.cpp [hunk 2]
```diff
@@ -1,8 +1,19 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * long with Vulpes.  If not, see <https://www.gnu.org/licenses/agpl-3.0.en.html>
  */
 
-spawn effect sig
-8B 4C 24 04 83 C8 FF 83 F9 FF 0F 84
+//spawn effect sig
+//8B 4C 24 04 83 C8 FF 83 F9 FF 0F 84
```
Comment (Mimickal) [inline vulpes/network/foxnet/effect.cpp]: danger

**Review by Mimickal — APPROVED** [review]
Approving this while pointing a little thing out

File: codegen/generate.pl (hunk 1 above)
Comment (ShellyHerself) [reply codegen/generate.pl:1]: Good catch, that was an accident

File: vulpes/network/foxnet/effect.cpp (hunk 2 above)
Comment (ShellyHerself) [reply vulpes/network/foxnet/effect.cpp]: I've debated just deleting this file tbh

Comment (ShellyHerself) [reply vulpes/network/foxnet/effect.cpp]: It's gone

**Pull request merged**